src/less/variables/ds6/typography-variables.less

```diff
@@ -10,4 +10,5 @@
 @font-size-28: 1.75rem;
 
 @font-weight-bold: 700;
+@font-weight-medium: @font-weight-bold;
```

Comment:
What is this for? I don't see it used here.

Reply:
@agliga In section-title, I use `font-size: @font-size-medium;`, which internally usees `@font-weight-bold`. And `font-weight-bold` was not defined in DS6.